src/functions-and-arrays.js

```diff
@@ -1,38 +1,68 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  } else {
+    return number2;
+  }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
-
+function findLongestWord(wordarray) {
+  if (wordarray.length === 0) {
+    return null;
+  }
+
+  let longestword = '';
+  for (let i = 0; i < wordarray.length; i++) {
+    if (wordarray[i].length > longestword.length) {
+      longestword = wordarray[i];
+    }
+  }
+  return longestword;
+}
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
-
-
+function sumNumbers(numbersarray) {
+  let result = 0;
+  for (let i = 0; i < numbersarray.length; i++) {
+    result = result + numbersarray[i];
```

Comment:
You could also write this as `result += numbersarray[i];` but this is just a tiny detail.